app/views/user_mailer/in_person_verified.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= image_tag(
+      'email/user-signup-ial2.png',
+      width: 140,
+      height: 177,
+      alt: '',
+      class: 'float-center padding-bottom-4',
+    ) %>
+<h4><%= message.subject %></h4>
+<p>
+<div class="s16">
+  <%= t('user_mailer.in_person_verified.greeting') %><br>
+  <%= t(
+        'user_mailer.in_person_verified.intro',
+        location: @presenter.location_name,
+        date: @presenter.formatted_verified_date,
+      ) %><br /><br />
+  <%= t('user_mailer.in_person_verified.next_sign_in') %>
+</div>
+</p>
+
+<div class="mt-40">
+<table class="button expanded large radius">
+  <tbody>
+    <tr>
+      <td>
+        <table>
+          <tbody>
+            <tr>
+              <td style="text-align: center">
+                <%= link_to t('user_mailer.in_person_verified.sign_in'),
+                            idv_url,
```

Comment:
where is idv_url defined?

Reply:
It's Rails magic; you can add `_url` to the end of the prefix of any defined route to get its URL. I wanted a link to `/verify`, so I ran
```bash
bundle exec rails routes
```
and looked to find that route in the output:
```
```

Reply:
oh that's cool